cypress/integration/learn/challenges/projects.js

```diff
@@ -64,7 +64,7 @@ describe('project submission', () => {
     () => {
       cy.fixture('../../config/curriculum.json').then(curriculum => {
         const { challenges, meta } =
-          curriculum[SuperBlocks.JsAlgoDataStruct].blocks[
+          curriculum['02-javascript-algorithms-and-data-structures'].blocks[
```

Comment:
Why can't the enum be used anymore?

Reply:
The real question is why does the `02-` need to be included now?

Reply:
It's part of my secret mission to put the number everywhere. While you're distracted by the numbers, I'll start sneaking in jQuery. 😁

Reply:
So, this is the one place (that I can find...) where we index the curriculum, other than during the build.  I've changed the build to stop stripping the `02-` etc off and instead just build an object like this

```js
var curriculum = {
"00-certifications": { blocks: [...] },
"01-responsive-web-design": { blocks: [...] },
"02-javascript-algorithms-and-data-structures": { blocks: [...] }
}
```

but that's only used in two places: here and during the build itself.  Everywhere else just wants a set of challenges and doesn't care how they're organised.

That is to say, this is a build artifact that we're abusing during testing, we could search for those challenges if we want.  Might be better, tbh, so if you want I'll refactor it.

Reply:
Erm... I would still like these magical numbers to disappear everywhere.

Reply:
Right. This decouples the dashed name of the superblock from the dir, so we're free to use more sensible names.

I don't have a strong opinion what names we should use, but this PR gives us that ability.